storagenode/storagenodedb/infodb.go

```diff
@@ -289,9 +289,47 @@ func (db *InfoDB) Migration() *migrate.Migration {
 				Description: "Clear Tables from Alpha data",
 				Version:     12,
 				Action: migrate.SQL{
-					`DELETE FROM pieceinfo`,
-					`DELETE FROM used_serial`,
-					`DELETE FROM order_archive`,
+					`DROP TABLE pieceinfo`,
+					`DROP TABLE used_serial`,
+					`DROP TABLE order_archive`,
+					`CREATE TABLE pieceinfo_ (
```

Comment:
I am not sure how the rename reduces the risk here since we are dropping the table?